frontend/pages/_app.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import App, { Container } from "next/app";
+import { ApolloProvider } from "react-apollo";
+import withApollo from "../lib/withApollo";
```

Comment:
single quotes pls

Reply:
Must be committed file before any prettier, eslint... setup :smile: